scipy/stats/stats.py

```diff
@@ -2516,6 +2517,116 @@ def iqr(x, axis=None, rng=(25, 75), scale='raw', nan_policy='propagate',
     return out
 
 
+
+def mad(x, axis=None, nan_policy='propagate', keepdims=False):
+    """
+    Compute the median absolute deviation (MAD) of the data along the
+    specified axis [1]_.
+
+    The MAD computes the median over the absolute deviations from the median.
+    It is a measure of dispersion similar to the standard deviation, but is
+    more robust to outliers [2]_.
+
+    The MAD of an empty array is `np.nan`.
+
+    .. versionadded:: 1.2
+
+    Parameters
+    ----------
+    x : array_like
+        Input array or object that can be converted to an array.
+    axis : int or sequence of int, optional
+        Axis along which the range is computed. The default is to
+        compute the IQR for the entire array.
```

Comment:
copy-paste error IQR.

Reply:
more importantly, better to make this `int or None`. the sequence of ints is annoying to implement and not necessary. this would solve your question on axis handling I think